packages/lesswrong/server/callbacks/dialogueMatchPreferenceCallbacks.ts

```diff
@@ -0,0 +1,197 @@
+import { getCollectionHooks } from "../mutationCallbacks";
+import { createNotifications } from "../notificationCallbacksHelpers";
+import { cheerioParse } from "../utils/htmlUtil";
+import { createAdminContext, createMutator, updateMutator } from "../vulcan-lib";
+
+interface MatchPreferenceFormData extends DbDialogueMatchPreference {
+  displayName: string;
+  userId: string
+}
+
+function getParagraphWithText(text: string) {
+  const $ = cheerioParse('<p></p>');
+  return $('p').text(text);
+}
+
+function convertTimestamp(timestamp: number) {
+  const date = new Date(timestamp);
+  return date.toUTCString();
+}
+
+const getDialogueMessageHTML = (userId:string, displayName:string, order:string, content:string ) => {
+
+  const time = convertTimestamp(Date.now())
+  const message_id = `${userId}-${time}`
+
+  const html = 
+    `<section class="dialogue-message ContentStyles-debateResponseBody" message-id="${message_id}" user-id="${userId}" display-name="${displayName}" submitted-date="${time}" user-order="${order}">
+      <section class="dialogue-message-header CommentUserName-author UsersNameDisplay-noColor"></section>
+      <div class="dialogue-message-content">
+        ${content}
+      </div>
+    </section>`
+
+  return html
+}
+
+const helperBotDisplayName = "Dialogue Helper Bot"
+const helperBotId = "edmzLyzymdoSuXnym"
```

Comment:
I'd put a todo about having this hardcoded

Reply:
This thing will actually break outside of dev, so needs to be fixed